lmfdb/templates/knowl-defs.html

```diff
@@ -39,18 +40,19 @@
  {% with hover = "%s [%s]"|format(ktitle, kid) -%}
   {% if title is none -%}
     {% if ktitle|length > 0 -%}
-     <a title="{{ hover }}" knowl="{{ kid }}" kwargs="{{ kwargs|urlencode }}">{{ ktitle|safe }}</a>
+     <a title="{{ hover }}" prefix="{{ kurlprefix }}" knowl="{{ kid }}" kwargs="{{ kwargs|urlencode }}">{{ ktitle|safe }}</a>
```

Comment:
I wonder if kurlprefix could just be a javascript variable set in the header of each page, that is accessed by lmfdb.js, that way the prefix won't be repeated for every knowl?